src/db_options.rs

```diff
@@ -136,18 +136,18 @@ impl Env {
             ffi::rocksdb_env_set_high_priority_background_threads(self.inner, n);
         }
     }
-
-    /// Wait for all threads started by StartThread to terminate.
-    pub fn join_all_threads(&mut self) {
+    
+    /// Sets the size of the bottom priority thread pool 
+    /// 
+    pub fn set_bottom_priority_background_threads(&mut self, n: c_int) {
```

Comment:
You call `ffi::rocksdb_env_set_high_priority_background_threads` in method which name includes `*bottom* ? 
If you look into C code than you'll see that this function has such code inside:
```cpp
void rocksdb_env_set_high_priority_background_threads(rocksdb_env_t* env, int n) {
  env->rep->SetBackgroundThreads(n, Env::HIGH);
}
```

Reply:
Thanks, I removed